orbit/cmd/desktop/desktop.go

```diff
@@ -221,7 +221,7 @@ func logDir() (string, error) {
 		if dir == "" {
 			return "", errors.New("$HOME is not defined")
 		}
-		dir += "/Library/Log"
+		dir += "/Library/Logs"
```

Comment:
nit: there's a comment above this function that also mentions `/Library/Log` 😮‍💨